### PR TITLE
5146 fix js console errors

### PR DESCRIPTION
**Pull request by alibombaye**

### CODE REVIEW

File: app/config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ module.exports = {
   appName: 'NHSD Buying Catalogue Admin',
 
   // Base URL
-  baseUrl: process.env.BASE_URL || '',
+  baseUrl: process.env.BASE_URL || '/admin',
```
Comment (kat1906) [inline app/config.js:6]: I think you need to change this back?

Comment (alibombaye) [reply app/config.js:6]: This is what it should be. When they run it in the cluster they use the defaults. We'll have to override it in our .env file.

But going via /admin should also work locally for you

Comment (kat1906) [reply app/config.js:6]: ok you will need to fix the tests then

Comment (kat1906) [reply app/config.js:6]: as IF you didn't run them before pushing 😆

Comment (alibombaye) [reply app/config.js:6]: haha normally I do but I thought ah the tests should be ok for this change. OK I might put it back and then do this later. Think it'll all be related to the relativeUrl stuff we still need to do for Admin

**Pull request merged**